cognite/transformations_cli/commands/utils.py

```diff
@@ -84,13 +85,15 @@ def print_query(query: str, result: TransformationPreviewResult) -> str:
     results = result.results
     if schema:
         print_res += "\nSchema:\n"
-        schema_content = [["name", "type", "nullable"]] + [[s.name, s.type.type, s.nullable] for s in schema]
+        schema_content = [["name", "type", "nullable"]] + [
+            [s.name, s.type.get("type", "") if isinstance(s.type, dict) else s.type.type, s.nullable] for s in schema
+        ]
```

Comment:
This works but actually, we should fix this in SDK, I'll take a look there.

Reply:
I don't think it's that bad. Some type conversions when using the sdk is not uncommon :)